test/functional/test.bat

```diff
@@ -17,5 +17,28 @@ for %%a in (%sites%) do (
     )
 )
 
+set sites_convert=test_site_convert
+
+for %%a in (%sites%) do (
+
+    echo(
+    echo Running %%a tests
+
+    node ../../index.js init %%a -c
+
+    node ../../index.js build %%a
+
+    node testUtil/test.js %%a "test_site_convert_expected"
+
+    rmdir /s /q %%a/_markbind
+    rmdir /s /q %%a/_site
+    del %%a/about.md %%a/index.md %%a/site.json
+
+    if errorlevel 1 (
```

Comment:
This is the wrong `errorlevel`; this `errorlevel` belongs to the `del` command rather than the `node testUtil/test.js` command. We want to test whether `node testUtil/test.js` failed, rather than whether `del` failed.